src/derp.c

```diff
@@ -198,35 +227,35 @@ static void derp_transmit_task(void *arg) {
 
 	// Prepare TLS configuration
 	esp_tls_cfg_t tls_cfg = {0};
-	tls_cfg.cacert_buf = cacert;
+	tls_cfg.cacert_buf = (unsigned char*) cacert;
 	tls_cfg.cacert_bytes = strlen(cacert) + 1;
 	tls_cfg.skip_common_name = true;
```

Comment:
Just a thought, that we should pass the `if_name` parameter to setting, to provide user to select the specifi interface to use for DERP connection. But this can be written of as a flex ... :smile:

Reply:
flex - :muscle: